.doc_gen/metadata/glacier_metadata.yaml

```diff
@@ -1,286 +1,340 @@
+  synopsis: upload an archive to an &GLlong; vault.
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/glacier
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - glacier.java2.upload.main
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/glacier
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - python.example_code.glacier.GlacierWrapper
+                - python.example_code.glacier.UploadArchive
+  services:
+    - glacier
+glacier_InitiateJob_InventoryRetrieval:
+  title: Retrieve an &GLlong; vault inventory using an &AWS; SDK
+  title_abbrev: Retrieve a vault inventory
+  synopsis: retrieve an &GLlong; vault inventory.
+  languages:
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/glacier
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - python.example_code.glacier.GlacierWrapper
+                - python.example_code.glacier.InitiateJob.InventoryRetrieval
+  services:
+    - glacier
+glacier_ListJobs:
+  title: List &GLlong; jobs using an &AWS; SDK
+  title_abbrev: List jobs
+  synopsis: list &GLlong; jobs.
+  languages:
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/glacier
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - python.example_code.glacier.GlacierWrapper
+                - python.example_code.glacier.ListJobs
+  services:
+    - glacier
+glacier_DeleteVault:
+  title: Delete an &GLlong; vault using an &AWS; SDK
+  title_abbrev: Delete a vault
+  synopsis: delete an &GLlong; vault.
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/glacier
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - glacier.java2.delete.main
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/glacier
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - python.example_code.glacier.GlacierWrapper
+                - python.example_code.glacier.DeleteVault
+  services:
+    - glacier
+glacier_InitiateJob_ArchiveRetrieval:
+  title: Retrieve an archive from an &GLlong; vault using an &AWS; SDK
+  title_abbrev: Retrieve an archive from a vault
+  synopsis: retrieve an archive from an &GLlong; vault.
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/glacier
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - glacier.java2.download.main

```

Comment:
This one should go under `glacier_InitiateJob_InventoryRetrieval`.

Reply:
fixed